### PR TITLE
Global styles: update return values from getGlobalStylesChanges()

### PR DESCRIPTION
**Pull request by ramonjd**
## What?
- Removes periods from return global style change items. 
- Remove "block" from the block labels, e.g., `"Audio"` vs `"Audio block"`

## Why?

Design feedback from:

- https://github.com/WordPress/gutenberg/pull/57470#issuecomment-1927066670

Also, let's leave it up to the consuming component as to how to present the changes. 

## Testing Instructions

1. In the site editor, make some global styles changes, making sure to update a block's global styles.
2. After clicking "Save", check that the global styles changes summary ends with a "period" and that "block" doesn't appear next to the block name.
3. Go ahead and save.
4. Check out the revision in the revisions panel.
5. Check that the revision's global styles changes summary ends with a "period" and that "block" doesn't appear next to the block name.

## Screenshots or screencast 
Screenshot 2024-02-06 at 1 33 38 pm

Screenshot 2024-02-06 at 1 33 49 pm

### CODE REVIEW

File: packages/edit-site/src/components/global-styles/screen-revisions/revisions-buttons.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ function ChangesSummary( { revision, previousRevision } ) {
 			data-testid="global-styles-revision-changes"
 			className="edit-site-global-styles-screen-revisions__changes"
 		>
-			{ changes.join( ', ' ) }
+			{ changes.join( ', ' ) }.
```
Comment (ramonjd) [inline packages/edit-site/src/components/global-styles/screen-revisions/revisions-buttons.js:39]: Pretty crude but it was the simplest approach after having tried the following:

- passing a "suffix" option value of `"."` to `getGlobalStylesChanges()`. Rejected because the function returns an array of strings and requires appending the suffix to the last item. Too weird.
- Changing the return value of `getGlobalStylesChanges()` to be a joined string. Rejected because it removes all the flexibility of a `string[]` return value.
- Creating a new function to return `changes.join( ', ' ) }.`. Rejected, or at least postponed, until we need it. We're only using `getGlobalStylesChanges()` in two places.

In summary, I'm of the opinion that the component should decide how to present the changes.

Comment (andrewserong) [reply packages/edit-site/src/components/global-styles/screen-revisions/revisions-buttons.js:39]: Since we're already using `,` to join the changes together, I think appending a `.` character here is likely fine!

**Review by andrewserong — APPROVED** [review]
Looking good! The period is now consistently being output and the `block` suffix has been removed 👍

image 
image
image
image

LGTM! ✨
